docs/digger/installation/macos-setup.asciidoc

```diff
@@ -55,5 +53,5 @@ ansible-playbook -i sample-inventory-file sample-playbook.yml
 
 ==== Further macOS Configuration options
 
-Further configuration of the macOS node is possible through optional variables.
-link:macos-options[View all macOS variables].
\ No newline at end of file
+xref:macos-variables[View all macOS variables].
```

Comment:
I think these are better the other way around. wdyt? make the comment and then provide the link?

Reply:
resolved